bouncer/src/repo/repo_controller_internal.cpp.inl

```diff
@@ -38,6 +39,29 @@ RepoController::_RepoControllerImpl::_RepoControllerImpl(
 	{
 		subscribeToLogger(listeners);
 	}
+
+	// set logging directory
+	std::string logDir = repo::lib::getEnvString("REPO_LOG_DIR");
+	std::string logPath = logDir.empty() ? "./log/" : logDir;
+	this->logToFile(logPath);
+
+	// set logging level
+	char* debug = getenv("REPO_DEBUG");
```

Comment:
should be using `getEnvString` here now you've created a function.

Please do a find around the codebase to find any `getnev` and replace them